packages/@lwc/babel-plugin-component/src/decorators/wire/transform.js

```diff
@@ -58,7 +69,8 @@ function getGeneratedConfig(t, wiredValue) {
                 t.identifier(varName),
                 t.memberExpression(
                     t.identifier(WIRE_CONFIG_ARG_NAME),
-                    t.identifier(memberExprPaths[0])
+                    memberExprPropertyGen(memberExprPaths[0]),
```

Comment:
same here.

Reply:
`memberExprPropertyGen` simply alternates the generation between dot notation (on well formed parameters) and bracket notation (malformed parameters)